sql/core/src/test/scala/org/apache/spark/sql/ExpressionsSchemaSuite.scala

```diff
@@ -156,14 +156,16 @@ class ExpressionsSchemaSuite extends QueryTest with SharedSparkSession {
       stringToFile(resultFile, goldenOutput)
     }
 
+    val outputSize = outputs.size
     val expectedOutputs: Seq[QueryOutput] = {
-      val goldenOutput = fileToString(resultFile)
-      val lines = goldenOutput.split("\n")
+      val expectedGoldenOutput = fileToString(resultFile)
+      val lines = expectedGoldenOutput.split("\n")
+      val expectedSize = lines.size
 
       // The header of golden file has one line, plus four lines of the summary and three
       // lines of the header of schema table.
-      assert(lines.size == outputs.size + 8,
-        s"Expected ${outputs.size + 8} blocks in result file but got ${lines.size}. " +
+      assert(expectedSize == outputSize + 8,
```

Comment:
@beliefer, can we have one string variable to keep the summary and header lines, and calculate the number of extra lines from the variable?

Reply:
OK.